Selenium.AutomatedTests/AutomationScenarioBuilder.cs

```diff
@@ -61,6 +66,7 @@ public AutomationScenarioBuilder WaitUntilVisible(By elementSelector)
         ///     executes <paramref name="action"/> if the element is located.
         /// </summary>
         /// <param name="elementSelector">Web element selector</param>
+        /// <param name="action"></param>
```

Comment:
`<param name="action">Action invoked with the located web element. It can be used for assertions and interactions.</param>`

Reply:
it is fixed in the commit 7f2820da5bb38f631308af8866df04ad9f81d2aa